src/utils/redux/requestIsSuccess.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * @flow
+ */
+
+import { RequestStates } from 'redux-reqseq';
+import type { RequestState } from 'redux-reqseq';
+
+export default function requestIsSuccess(requestState :?RequestState) :boolean {
```

Comment:
We already know we're working with request states. Please rename to `isSuccess`.